### PR TITLE
Update docs to better reflect recent schema changes

### LINKED ISSUE #666: Update docs to reflect new schema
https://github.com/responsible-ai-collaborative/aiid/blob/staging/mongo.md

### PR DESCRIPTION
**Pull request by cesarvarela**
Closes #666 part of #524

### CODE REVIEW

File: mongo.md [hunk 1]
```diff
@@ -10,25 +10,50 @@ Administering data requires administrative access to the database. This access i
...
+
+* `editors`: ["Anonymous"] # (String[]) Editors responsible for the Incident
+
+### Text
+
+* `title`: ["Anonymous"]: String title
+
+### Other
+
+* `Alleged deployer of AI system`: String[] ??
```
Comment (cesarvarela) [inline mongo.md:43]: @Sean please help with these

File: mongo.md [hunk 2]
```diff
@@ -39,37 +64,38 @@ Dates
...
 * `authors`: Array(string) # People that wrote the incident report
 * `editors`: Array(string) # The editors of the incident. Editors are listed in the incident citation and can promote user-submitted records to incident reports.
 
-Text
+### Text
 
 * `title`: "title of the report" # (string) The title of the report that is indexed.
 * `description`: "Short text for the report"
 * `text`: "Long text for the report" # (string) This is the complete text for the report in the MongoDB instance, and a shortened subset in the Algolia index
+* `tags`: ["Some Tag"] # (String[]) Tags assoaciated with the report.
```
Comment (smcgregor) [inline mongo.md:78]: `Tags assoaciated with the report.` -> `Tags associated with the report.`

File: mongo.md (hunk 1 above)
Comment (smcgregor) [reply mongo.md:43]: `Alleged deployer of AI system`: String[] The entity responsible for deploying the AI system in the real world.

File: mongo.md [hunk 3]
```diff
@@ -10,25 +10,50 @@ Administering data requires administrative access to the database. This access i
...
+* `editors`: ["Anonymous"] # (String[]) Editors responsible for the Incident
+
+### Text
+
+* `title`: ["Anonymous"]: String title
+
+### Other
+
+* `Alleged deployer of AI system`: String[] ??
+* `Alleged developer of AI system`: String[] ??
```
Comment (smcgregor) [inline mongo.md:44]: `Alleged developer of AI system`: String[] The entities responsible for developing the AI system in the real world.

File: mongo.md [hunk 4]
```diff
@@ -10,25 +10,50 @@ Administering data requires administrative access to the database. This access i
...
+* `Alleged harmed or nearly harmed parties`: String[] ??
```
Comment (smcgregor) [inline mongo.md:45]: `Alleged harmed or nearly harmed parties`: String[] The entities harmed as a result of the AI system.

File: mongo.md [hunk 5]
```diff
@@ -10,25 +10,50 @@ Administering data requires administrative access to the database. This access i
...
-* `ref_number`: 25 # (int) The reference number scoped to the incident ID.
+* `reports`: [1, 2, 3] # (Int[]) The list of reports by `report_number`, that belong to this Incident
+
+### Dates
+
+* `date`: 2019-07-25 # (String) Date the incident occurred.
+
+### People
+
+* `editors`: ["Anonymous"] # (String[]) Editors responsible for the Incident
```
Comment (smcgregor) [inline mongo.md:35]: editors are never anonymous. Right now it defaults to "Sean McGregor".

**Comment by smcgregor** [comment]
A few tweaks, otherwise good.

**Comment by cesarvarela** [comment]
Updated

**Pull request merged**